examples/simple_script_vendored/WORKSPACE

```diff
@@ -10,15 +10,9 @@ local_repository(
     path = "../..",
 )
 
-load(
-    "@rules_ruby//ruby:deps.bzl",
-    "rules_ruby_dependencies",
-    "rules_ruby_select_sdk",
-)
-
+load("@rules_ruby//ruby:deps.bzl", "rules_ruby_dependencies", "rules_ruby_register_toolchains")
 rules_ruby_dependencies()
-
-rules_ruby_select_sdk(version = "2.7.1")
+rules_ruby_register_toolchains(["ruby-2.7.1"])
```

Comment:
Should this stay `2.7.1`? Is there a value in pinning this to a particular version number in the middle of our supported versions range?

Reply:
This is just an example used for testing.  The specific version used here is irrelevant